Gemfile

```diff
@@ -13,6 +13,7 @@ gem "jquery-rails"
 gem "neat", "~> 1.7.0"
 gem "normalize-rails"
 gem "paper_trail"
+gem "paperclip", "~> 5.0.0"
```

Comment:
@derekprior, I could not remember which version of Paperclip you updated to so I just copied the one in the Paperclip documentation. Happy to `bundle install` a different version.